app/docs/dev/policies/meshtrafficpermission.md

```diff
@@ -0,0 +1,280 @@
+---
+title: MeshTrafficPermission (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficPermission](../traffic-permissions).
+{% endwarning %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+| ----------------- | --------- | --- | ---- |
+| Mesh              | ✅        | ❌  | ✅   |
+| MeshSubset        | ✅        | ❌  | ✅   |
+| MeshService       | ✅        | ❌  | ✅   |
+| MeshServiceSubset | ✅        | ❌  | ✅   |
+| MeshGatewayRoute  | ✅        | ❌  | ❌   |
```

Comment:
If there is no to/from then how is `MeshGatewayRoute` configured?

Reply:
I think there is no way to configure it by [looking at the code](https://github.com/kumahq/kuma/blob/db9de2b90fabf25c4e146d77120fc3889cb98612/pkg/plugins/policies/meshtrafficpermission/api/v1alpha1/meshtrafficpermission.go#L8-L17).

Reply:
Yeah, I think I had the same problem reading this matrix at first, like "if the top level is MeshGatewayRoute then to/from can't be set". While it should be read like "MeshGatewayRoute could be used only in the top level"

Reply:
I didn't understand the question initially, @slonka is right, today there is no way to use MeshTrafficPermission with MeshGatewayRoute. In the future, we'll be able to add this but only for cross-mesh. So I removed MeshGatewayRoute from matrix